proto/p4/v1/p4runtime.proto

```diff
@@ -284,6 +284,18 @@ message Action {
 
 message ActionProfileActionSet {
   repeated ActionProfileAction action_profile_actions = 1;
+  enum BucketSelectionMode {
```

Comment:
Please add comments explaining the use case of this. Assumably, it's to support WCMP